src/MoBi.Presentation/Tasks/ObservedDataTask.cs

```diff
@@ -165,6 +167,15 @@ private static MoBiMacroCommand deleteAllResultsFromSimulationCommand(IMoBiSimul
          return macroCommand;
       }
 
+      public void DeleteResultsFromSimulationCommand(IMoBiSimulation simulation, DataRepository dataRepository)
```

Comment:
Rename to drop `Command` from the name. The task is using a command, but the action is `DeleteResultsFromSimulation`